langroid/agent/special/doc_chat_agent.py

```diff
@@ -405,6 +419,10 @@ def ingest_docs(
         if self.vecdb is None:
             raise ValueError("VecDB not set")
 
+        if self.config.hypothetical_questions:
+            question_docs = [self.llm_hypothetical_question(doc) for doc in docs]
```

Comment:
This will be very slow when there are hundreds/thousands of chunks. We should use an appropriate batch method, see `langroid/agent/batch.py`

Reply:
done